tasks/auth.py

```diff
@@ -18,6 +19,13 @@ def datadog_infra(ctx, audience, datacenter="us1.ddbuild.io"):
     print(token)
 
 
+@task
+def gitlab(ctx, repo='datadog-agent', verbose=False):
+    """Get a gitlab token."""
+
+    print(get_gitlab_token(ctx, repo, verbose))
```

Comment:
```suggestion
    if running_in_ci():
        raise Exit(message='This task is meant to be run locally, not in CI', code=1)
    
    if "GITLAB_TOKEN" in os.environ:
        print(os.environ["GITLAB_TOKEN"])
    else:
        print(get_gitlab_token(ctx, repo, verbose))
```
To be consistent with the GITHUB task